### PR TITLE
add a issue template for membership request

### PR DESCRIPTION
**Pull request by Iceber**
I'm requesting to be a Member of Clusterpedia Community.

### Others

### CODE REVIEW

**Comment by Iceber** [comment]
@haifeng-yao @pacoxu @wzshiming @calvin0327 PTAL, Thanks

File: .github/ISSUE_TEMPLATE/membership-request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+I'm requesting to be a Member of Clusterpedia Community.
+
+## List of my contributions to the Clusterpedia project
+> As a Member, we expect you to have merged at least three pr.
+- PRs authored
+
+- PRs reviewd
```
Comment (pacoxu) [inline .github/ISSUE_TEMPLATE/membership-request.md:16]: reviewed

Comment (Iceber) [reply .github/ISSUE_TEMPLATE/membership-request.md]: updated, Thanks

File: .github/ISSUE_TEMPLATE/membership-request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: "[MEMBERSHIP REQUEST] New Member of Clusterpedia"
+labels: kind/membership
+assignees: Iceber
+
+---
+
+I'm requesting to be a Member of Clusterpedia Community.
+
+## List of my contributions to the Clusterpedia project
+> As a Member, we expect you to have merged at least three pr.
```
Comment (wzshiming) [inline .github/ISSUE_TEMPLATE/membership-request.md:13]:
```suggestion
<!--
As a Member, we expect you to have merged at least three pr.
-->
```

File: .github/ISSUE_TEMPLATE/membership-request.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- PRs authored
+
+- PRs reviewed
+
+- Issues responded to and created
+
+- Others
```
Comment (wzshiming) [inline .github/ISSUE_TEMPLATE/membership-request.md:21]:
```suggestion
### PRs authored

### PRs reviewed

### Issues responded to and created

### Others


```

File: .github/ISSUE_TEMPLATE/membership-request.md (hunk 2 above)
Comment (Iceber) [reply .github/ISSUE_TEMPLATE/membership-request.md:13]: nice 👍🏻

File: .github/ISSUE_TEMPLATE/membership-request.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+I'm requesting to be a Member of Clusterpedia Community.
+
+## List of my contributions to the Clusterpedia project
+### PRs authored
+<!--
+As a Member, we expect you to have merged at least three pr.
+-->
+### PRs reviewed
+### Issues responded to and created
```
Comment (windsonsea) [inline .github/ISSUE_TEMPLATE/membership-request.md:18]: `Issues responded to and created` seems relatively longer, how about `Issues involved with`

Comment (Iceber) [reply .github/ISSUE_TEMPLATE/membership-request.md:18]: Compared to `PRs authored` and `PRs reviewed`, it is indeed a bit longer.
@pacoxu @wzshiming Do you find `Issues involved` a bit obscure and could replace `Issues responded to or created` ?

Comment (Iceber) [reply .github/ISSUE_TEMPLATE/membership-request.md]: @windsonsea used `Issues involved`, and added some notes

Comment (windsonsea) [reply .github/ISSUE_TEMPLATE/membership-request.md]: /lgtm

**Pull request merged**